src/Native/Unix/System.IO.Ports.Native/pal_termios.c

```diff
@@ -0,0 +1,482 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "pal_types.h"
+#include <termios.h>
+#include <sys/ioctl.h>
+#include <unistd.h>
+#include <errno.h>
+#include <pal_termios.h>
+
+/* This is dup of System/IO/Ports/NativeMethods.cs */
+enum
+{
+    ParityNone = 0,
+    ParityOdd  = 1,
+    ParityEven = 2,
+    ParityMark = 3,
+    ParitySpace = 4
+};
+
+/* System/IO/Ports/Handshake.cs */
+enum
+{
+    HandshakeNone,          // No Flow Control
+    HandshakeSoft,          // Software Flow Control -> XOn/Xoff
+    HandshakeHard,          // Hardware Flow Control -> RTS/CTS
+    HandshakeBoth           // Software & Hardware Flow Control
+};
+
+/* Interop/Unix/Interop.Termios.cs */
+enum {
+    SignalDtr = 1,
+    SignalDsr = 2,
+    SignalRts = 3,
+    SignalCts = 4,
+    SignalDcd = 5,
+};
+
+enum {
```

Comment:
A nit - can you please put the curly brace on a separate line like in the previous enums?